geode-web-management/src/integrationTest/java/org/apache/geode/management/internal/rest/MemberManagementServiceRestIntegrationTest.java

```diff
@@ -91,7 +91,9 @@ public void listServer() throws Exception {
         .andExpect(jsonPath("$.result[0].locator", is(false)))
         .andExpect(jsonPath("$.result[0].cacheServers[0].port", greaterThan(0)))
         .andExpect(jsonPath("$.result[0].cacheServers[0].maxConnections", equalTo(800)))
-        .andExpect(jsonPath("$.result[0].cacheServers[0].maxThreads", equalTo(0)));
+        .andExpect(jsonPath("$.result[0].cacheServers[0].maxThreads", equalTo(0)))
+        .andExpect(jsonPath("$.result[0].groups", containsInAnyOrder("group-1", "group-2")))
+        .andExpect(jsonPath("$.result[0].groups[2]").doesNotExist());
```

Comment:
I believe this line is unnecessary. the `containsInAnyOrder` already confirms that the groups should only have 2 items. it's like `containsExactlyInAnyOrder` according to the javadoc.

Reply:
Oh neat. Thank you!